scico/optimize/admm.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, minimize_kwargs: dict = {"options": {"maxiter": 100}}):
                 :func:`scico.solver.minimize`.
         """
         self.minimize_kwargs = minimize_kwargs
+        self.info = {}
```

Comment:
Yes, it is inconsistent, but I thought this was justified by the different circumstances. In `GenericSubproblemSolver`, it has to be initialized as a dict because of the mechanism used to copy in attributes from the optimization result object. In `LinearSubproblemSolver`, on the other hand, the info is directly returned in the form of a dict, so initializing to `None` seemed reasonable. But I guess we could also switch this to an empty dict if you think the inconsistency is a significant issue.

Reply:
Something good is that this works in both cases. If we somehow want to test for:

```
if self.info:
    #do something
```

the if statement picks em up as false so its all good by me.

Reply:
OK, good.